experiment_helpers/check_rmsThresh.m

```diff
@@ -55,9 +55,15 @@
             rmsValue = mean(data.rms(onset:offset, 1));
 
         % if no ost tracking, use RMS data to find onset/offset
-        elseif ~any(data.ost_stat >= 1) && any(data.rms(:, 1) > 0.03)
-            onset = find(data.rms > 0.01, 1, 'first') + 5;
-            offset = find(data.rms(:, 1)<0.03 & data.rms(:, 1)>0.02 & data.rms_slope<0, 1, 'first') - 5;
+        elseif any(data.rms(:, 1) > 0.025)
+            onset = find(data.rms > 0.025, 1, 'first') + 5;
+            offset = find(data.rms(:, 1)<0.03 & data.rms(:, 1)>0.015 & data.rms_slope<0, 1, 'first') - 5;
```

Comment:
Some of the values (0.025, 0.03, 0.015, etc.) changed -- how were they determined? Can we use variables / comment what they correspond to?

Reply:
The values are originally based on the same used in measureFormants.ost. Basically, trying to use the same behavior with data.rms that Audapter uses to calculate OST values, to create parity between the OST-based and non-OST-based methods for finding onset/offset.

In general, I need to add more intricacy to the non-OST-based method to make it more consistent and more in-line with how the OSTs work. However, Robin recently recommended that we change the heuristics used to calculate OSTs in measureFormants.ost. I want to test that out, and assuming it's good, fine-tune the non-OST-based RMS calculation in this function.

After running the participant this afternoon, I think lowering these values actually might cause more problems than it fixes, so I will revert this change. I can also add a comment saying that these values are based on measureFormants.ost

I like the idea of having the non-OST-based calculation be more configurable via fields in expt.amplcalc -- that was my long-term plan. I think that should be in a separate PR. This change maybe shouldn't have been in this PR to start with.

Reply:
Agree with making a separate PR to tune these values. I notice one is still different (onset used to be 0.01, now 0.03) -- is this intentional?